lib/install.js

```diff
@@ -784,6 +787,26 @@ Installer.prototype.printInstalledForHuman = function (diffs, cb) {
       ++moved
     } else if (mutation === 'add') {
       ++added
+      // Count contributors to added packages. Start by combining `author`
+      // and `contributors` data into a single array of contributor-people
+      // for this package.
+      var people = []
+      var meta = pkg.package
+      if (meta.author) people.push(meta.author)
+      if (meta.contributors && Array.isArray(meta.contributors)) {
+        people = people.concat(meta.contributors)
+      }
+      // Make sure a normalized string for every person behind this
+      // package is in `contributors`.
+      people.forEach(function (person) {
+        // Ignore errors from malformed `author` and `contributors`.
```

Comment:
Under what circumstances could `normalizePerson` throw? To my reading it would only throw if `person` were `undefined` or `null`? If that's the case, then I think it would be better to just skip those cases…

Reply:
I didn't have a case in mind. Just `try`'d it out of prudence.

Perhaps a `TypeError` on String coercion, if `person.name` or similar is an Object?